app/src/main/java/com/edulectronics/tinycircuit/Models/MessageArgs.java

```diff
@@ -9,20 +9,19 @@
 public class MessageArgs implements Serializable{
     public MessageTypes type;
     public int title;
-    public int message;
+    public String message;
 
     public boolean endActivity;
     public boolean goToNextScenario;
 
-    public MessageArgs(int message, MessageTypes type) {
+    public MessageArgs(String message, MessageTypes type) {
```

Comment:
Het nieuwe feedback systeem maakt gebruik van een string array. De items van een string array hebben een eigen id, dus is het onmogelijk om een id te gebruiken.